config/hpe.conf.sample.3par

```diff
@@ -50,6 +50,7 @@ san_ip = <SAN IP>
 san_login = <SAN username>
 san_password = <SAN password>
 hpe3par_cpg = <CPG name used for volume creation>
+hpe3par_snapcpg = <CPG name used for snapshot creation>
```

Comment:
Please update the iSCSI, FC hpe.conf templates in 
- https://github.com/hpe-storage/python-hpedockerplugin/blob/master/config/hpe.conf.sample.3parISCSI
- https://github.com/hpe-storage/python-hpedockerplugin/blob/master/config/hpe.conf.sample.3parFC